metrics_api/lib/opentelemetry/metrics/meter.rb

```diff
@@ -55,19 +83,87 @@ def create_histogram(name, unit: nil, description: nil)
       def create_gauge(name, unit: nil, description: nil)
         create_instrument(:gauge, name, unit, description, nil) { GAUGE }
       end
-
+      # UpDownCounter is a synchronous Instrument which supports increments and decrements.
+      #
+      # With this api call:
+      #
+      #   items_counter = meter.create_up_down_counter("store.inventory",
+      #                                                description: "the number of the items available",
+      #                                                unit: "s")
+      #
+      # @param name [String] the name of the up_down_counter
+      # @param unit [optional String] an optional string provided by user.
+      # @param description [optional String] an optional free-form text provided by user.
+      #
+      # @return [nil] after creation of up_down_counter, it will be stored in instrument_registry
```

Comment:
```suggestion

      # UpDownCounter is a synchronous Instrument which supports increments and decrements.
      #
      # With this api call:
      #
      #   items_counter = meter.create_up_down_counter("store.inventory",
      #                                                description: "the number of the items available",
      #                                                unit: "s")
      #
      # @param name [String] the name of the up_down_counter
      # @param unit [optional String] an optional string provided by user.
      # @param description [optional String] an optional free-form text provided by user.
      #
      # @return [nil] after creation of up_down_counter, it will be stored in instrument_registry
```